ci/pipelines/builder.yml

```
@@ -807,6 +841,13 @@ resources:
   source:
     initial_version: true
     interval: 24h
+- name: weekly
+  type: time
+  source:
+    start: 3:00 -0700
+    stop: 4:30 -0700
+    days: [ Saturday ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)